models/Book.js

```diff
@@ -1,13 +1,17 @@
-"use strict";
+import mongoose from 'mongoose';
+import mockgoose from 'mockgoose';
+if (process.env.NODE_ENV === 'test') {
+  mockgoose(mongoose);
+}
+const mongodbCredential = process.env.KINDLIZED_MONGODB;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).